lib/diagnostic.js

```diff
@@ -32,6 +52,13 @@ const newArray = function newArray(/* length, defaultValue */) {
 // to the result of invoking defaultValue with the
 // element's index as an argument
 
+const newArray = function newArray(){
```

Comment:
```newArray``` was already defined earlier by using ```const``` so trying to redefine or reassign it will result in an error.